config/routes.rb

```diff
@@ -24,6 +24,30 @@
     get '/', to: 'inthenews#home', as: :inthenews_home,
       defaults: { section: "inthenews" }
 
+    # Wireframes
+    get '/poets', to: 'inthenews#poets', as: :inthenews_poets,
+      defaults: { section: "inthenews" }
+    get '/events', to: 'inthenews#events', as: :inthenews_events,
+      defaults: { section: "inthenews" }
+    get '/news', to: 'inthenews#news', as: :inthenews_news,
+      defaults: { section: "inthenews" }
+    get '/relationships', to: 'inthenews#relationships', as: :inthenews_relationships,
+      defaults: { section: "inthenews" }
+    get '/about', to: 'inthenews#about', as: :inthenews_about,
+      defaults: { section: "inthenews" }
+
+    get '/poets/wolesoyinkamain', to: 'inthenews#wolesoyinkamain', as: :inthenews_wolesoyinkamain,
```

Comment:
same comment as above for the URL structure

Reply:
Leave for now